doc/src/aboutus.rst

```diff
@@ -24,6 +24,7 @@ Financial and Infrastructure Support
   * in 2012 for 6 students (`GSoC 2012 <https://github.com/sympy/sympy/wiki/Gsoc-2012-report>`_)
   * in 2013 for 7 students (`GSoC 2013 <https://github.com/sympy/sympy/wiki/Gsoc-2013-report>`_)
   * in 2014 for 10 students (`GSoC 2014 <https://github.com/sympy/sympy/wiki/Gsoc-2014-report>`_)
+  * in 2016 for 7 students (`GSoC 2016 <https://github.com/sympy/sympy/wiki/GSoC-2016-report>`_)
```

Comment:
Maybe this top list should have complete totals (e.g., 9 for this year), and then below it, we can write "of these, we would like to thank these organizations for hosting GSoC students under their umbrella organizations:"